settings.gradle.kts

```diff
@@ -0,0 +1,21 @@
+@file:Suppress("UnstableApiUsage")
+
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        maven("https://api.xposed.info")
+        mavenCentral()
+        maven("https://jitpack.io")
```

Comment:
Do we need https://jitpack.io? Seems https://api.xposed.info contains the packages we need.

Reply:
You're right, I have removed it :)